src/components/DatabaseChanges.tsx

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+import * as React from 'react';
+
+import { a as A, p as P } from './Mdx';
+
+interface IDatabaseName {
+  product: string;
+}
+
+const DatabaseChanges: React.FC<IDatabaseName> = (props) => {
+  const { product } = props;
+  return (
+    <>
+      <P>
+        We may add new data fields to the
+        {' '}
+        {product}
+        {' '}
+        database at any time.
+      </P>
+
+      <P>
+        New database fields are added as new columns to the right of existing
+        {' '}
+        columns in our CSV files, and as additional data in our MMDB files.
+      </P>
+
+      <P>
+        Subscribe to our
+        {' '}
+        <A
+          href="https://dev.maxmind.com/geoip/release-notes?lang=en"
```

Comment:
A relative link would work better here.  Also, do we really need to open the link in a new window?